src/Stratis.Bitcoin/P2P/PeerConnectorDiscovery.cs

```diff
@@ -91,14 +92,6 @@ public override async Task OnConnectAsync()
                     continue;
                 }
 
-                // If the peer is already connected just continue.
-                if (this.IsPeerConnected(peer.Endpoint))
```

Comment:
Why remove this?

Reply:
This is now checked in the base class :)